text-ui-test/EXPECTED.TXT

```diff
@@ -5,3 +5,39 @@ Hello from
 | |_| | |_| |   <  __/
 |____/ \__,_|_|\_\___|
 
+What can I do for you?
```

Comment:
Great to see that you are doing tests! Your test-driven development greatly helps to reduce the number of bugs that you might run into in the future, which can save time. Great job!